jobs/tests/test_inventory_to_records.py

```diff
@@ -634,3 +635,12 @@ def tearDown(self):
                 }
             }
         }
+
+    @mock.patch('buildhub.utils.guess_mimetype',
```

Comment:
Try to mock guess_mimetype directly in the inventory_to_record file.

Reply:
`guess_mimetype` is only used in `utils`